conf/views/waitOnGuaranteeBalance.njk

```diff
@@ -13,12 +13,18 @@
     <div class="govuk-grid-column-two-thirds">
 
       <h1 class="govuk-heading-xl">{{ messages("waitOnGuaranteeBalance.heading") }}</h1>
-      <p class="govuk-body">{{ messages("waitOnGuaranteeBalance.paragraph", waitTimeInSeconds) }}</p>
+      <p class="govuk-body">{{ messages("waitOnGuaranteeBalance.checkDetails.prelink") }}
+        <a class="govuk-link" id="checkDetails-link" href="{{ routes.controllers.CheckYourAnswersController.onPageLoad().url }}">
```

Comment:
I would consider making the case of the id consistent. `check-details` or `recheck-details` would probably suffice. Otherwise looks good to me